guides/common/modules/proc_synchronizing-ansible-collections.adoc

```diff
@@ -23,7 +20,7 @@ The *Label* field is populated automatically based on the name.
 . In the *Upstream URL* field, enter the URL for the upstream collections repository.
 +
 The URL can be any Galaxy endpoint.
-For example, `++https://cloud.redhat.com/api/automation-hub/++`.
+For example, `++https://console.redhat.com/api/automation-hub/++`.
```

Comment:
```suggestion
For example, `https://console.redhat.com/api/automation-hub/`.
```

`++` is required if links contain spaces or special characters: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#links

Here, it's IMHO not needed.